tsconfig.json

```diff
@@ -51,7 +51,7 @@
       "huaweicloud": ["huaweicloud/src"]
     },
     "pretty": true,
-    "removeComments": true,
+    "removeComments": false,
```

Comment:
should we leave a 'do not change this or you will be fired' comment here?

Reply:
its a json file, so comments aren't supported, but I agree with the idea 😅

Reply:
there are [comments](https://github.com/spinnaker/deck/pull/8180/files/1c59e5dba065230a56410d51d79349956dbce6aa#diff-e5e546dd2eb0351f813d63d1b39dbc48R58) in every json file in this PR :)

Reply:
`tsconfig.json` is _"special"_ json

Reply:
ahhh TIL! :)